local/yace/config/config.yml

```diff
@@ -253,6 +303,35 @@ static:
     customTags:
       - key: Environment
         value: localstack
+  - namespace: AWS/S3
```

Comment:
I don’t think this static config is required

Reply:
Without static config - only all_request metric coming through that is after couple of minutes;

```
# HELP aws_s3_all_requests_sum Help is not implemented yet.
# TYPE aws_s3_all_requests_sum gauge
aws_s3_all_requests_sum{account_id="000000000000",name="global",region="eu-west-2",tag_CreatedBy="",tag_Environment="",tag_Team=""} 150 1668797880000
```

With static config - data coming through straight away in couple of seconds;

```
# HELP aws_s3_4xx_errors_sum Help is not implemented yet.
# TYPE aws_s3_4xx_errors_sum gauge
aws_s3_4xx_errors_sum{account_id="000000000000",name="my-id",region="eu-west-2"} 10
# HELP aws_s3_5xx_errors_sum Help is not implemented yet.
# TYPE aws_s3_5xx_errors_sum gauge
aws_s3_5xx_errors_sum{account_id="000000000000",name="my-id",region="eu-west-2"} 10
# HELP aws_s3_all_requests_sum Help is not implemented yet.
# TYPE aws_s3_all_requests_sum gauge
aws_s3_all_requests_sum{account_id="000000000000",name="my-id",region="eu-west-2"} 250
# HELP aws_s3_total_request_latency_average Help is not implemented yet.
# TYPE aws_s3_total_request_latency_average gauge
aws_s3_total_request_latency_average{account_id="000000000000",name="my-id",region="eu-west-2"} 1
```